configure.ac

```diff
@@ -112,8 +112,18 @@ AC_DEFINE_UNQUOTED([UCX_CONFIGURE_FLAGS], ["$config_flags"], [UCX configure flag
 #
 # Define path of ucx.conf configuration file
 #
+case "$prefix" in
+    NONE)
+        case "$sysconfdir" in
+            '${prefix}/etc')
+                sysconfdir=/etc # Prevent sysconfdir from being "NONE/etc"
+                ;;
+        esac
+        ;;
+esac
+# Evaluate the '${prefix}/etc' expression
```

Comment:
seems weird
we should take config file name from config.h which is the substituted version of config.h.in

Reply:
Can elaborate please? `config.h.in` is generated by this `configure.ac`.
I defined `UCX_CONF_FILE` here in a manner similar to how `UCX_MODULE_SUBDIR` is defined below, for example.

Where should we define `UCX_CONF_FILE` instead?